lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -417,6 +444,7 @@ defmodule ExUnit.DocTestTest do
   doctest ExUnit.DocTestTest.IndentationHeredocs
   doctest ExUnit.DocTestTest.FencedHeredocs
   doctest ExUnit.DocTestTest.Haiku
+  doctest ExUnit.DocTestTest.PatternMatching, only: [tuple: 0, map: 0], import: true
```

Comment:
These two functions that did have success cases in their doctests now don't exist, so we can probably remove this line since it does nothing. In fact, I'm surprised it doesn't raise an exception...